extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeMixPlaylistExtractor.java

```diff
@@ -89,7 +89,7 @@ public void onFetchPage(@Nonnull final Downloader downloader)
         final byte[] body = JsonWriter.string(jsonBody.done()).getBytes(StandardCharsets.UTF_8);
 
         final Map<String, List<String>> headers = new HashMap<>();
-        addClientInfoHeaders(headers);
+        addYouTubeHeaders(headers);
```

Comment:
`addYouTubeHeaders` adds the consent cookie in addition to calling `addClientInfoHeaders`